salt/haproxy/haproxy.cfg.jinja

```diff
@@ -59,6 +59,8 @@ backend no-timeout-backend
 
 
 {% if "admin" in salt['grains.get']('roles', []) %}
+errorfile 503 /etc/caasp/haproxy/errors/503_database_not_ready.html
```

Comment:
@ereslibre it might be that the file needs to end with `.http` I am not really sure there

Reply:
@MaximilianMeister  @ereslibre The doc says It is recommended to follow the common practice of appending ".http" to the filename so that people do not confuse the response with HTML error pages so it should work with .html

https://cbonte.github.io/haproxy-dconv/1.6/configuration.html#4-errorfile

Reply:
It looks like we need to set `.http` as the response, but also include the headers of the response. What's going to be returned here is the full contents of this file as the HTTP response on the TCP socket, so we need something like:

```
HTTP/1.0 503 Service Unavailable
Cache-Control: no-cache
Connection: close
Content-Type: text/html

<contents here>
```

Reply:
i've submitted https://github.com/kubic-project/velum-branding/pull/6 to fix this. also from reading the docs, i had to minify the html files, otherwise haproxy would truncate them after reaching 16kb :laughing: